api/hub/src/helpers.ts

```diff
@@ -209,6 +210,32 @@ export const sendAuthorNotification = async (
   return sendNotification(recipient, notification);
 };
 
+/**
+ * Send an email notification using SendGrid.
+ * @param to - Destination email address
+ * @param from - Source email address
+ * @param subject - Email subject
+ * @param text - Email body (text version)
+ * @param html - Email body (HTML version)
+ * @returns A promise that resolves upon sending the email
+ */
+export const sendEmailNotification = async (
+to: string,
```

Comment:
consider using an object param with these attributes, makes the function signature easier to read when it is called